tests/e2e/plugin_workflow.ipynb

```diff
@@ -0,0 +1,309 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "7cb3af19",
+   "metadata": {},
+   "source": [
+    "# End to End test of the plugin workflow"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e5c95c03",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from kili.client import Kili\n",
+    "from pathlib import Path\n",
+    "import os\n",
+    "from cuid import cuid"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1d62f861",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/paulm/Documents/Git/GitHub_Kili/kili-playground/src/kili/authentication.py:128: UserWarning: \n",
+      "Your api key will be deprecated on 2023-02-16.\n",
+      "You should generate a new one on My account > API KEY.\n",
+      "  warnings.warn(message, UserWarning)\n"
+     ]
+    }
+   ],
+   "source": [
+    "kili = Kili(\n",
+    "    api_endpoint=\"https://staging.cloud.kili-technology.com/api/label/v2/graphql\",\n",
+    "    api_key=os.getenv(\"KILI_ADMIN_API_KEY_STAGING\"),\n",
+    ")"
```

Comment:
I think you can just use `kili = Kili() `and in the e2e tests there are the proper variables for api_key and endpoint

Reply:
yes, you're right